sdk/cognitiveservices/arm-cognitiveservices/README.md

```diff
@@ -19,8 +19,9 @@ npm install @azure/arm-cognitiveservices
 
 ##### Install @azure/ms-rest-nodeauth
 
+- Please install minimum version of `"@azure/ms-rest-nodeauth": "^3.0.0"`.
 ```bash
-npm install @azure/ms-rest-nodeauth
+npm install @azure/ms-rest-nodeauth@"^3.0.0"
```

Comment:
`npm install` command installs the latest version always. Therefore, we dont need to specify the version this way

Reply:
Same as above